tests/util.ts

```diff
@@ -0,0 +1,6 @@
+import PeerId from 'peer-id'
+import { getIdentity } from './identities'
+
+export async function peerIdForIdentity(identityName: string): Promise<PeerId> {
+  return PeerId.createFromPrivKey(getIdentity(identityName))
```

Comment:
This method is sometimes bit dangerous as it expects libp2p-specific padding, therefore we have the method https://github.com/hoprnet/hoprnet/blob/master/packages/utils/src/libp2p/privKeyToPeerId.ts